internal/model/plugin.go

```diff
@@ -17,7 +17,8 @@ type Plugin struct {
 	Labels          []Label                   `json:"labels,omitempty"`
 	Signature       string                    `json:"signature"` // A signature of a plugin saved in base64 encoding.
 	Manifest        *mattermostModel.Manifest `json:"manifest"`
-	UpdatedAt       time.Time                 `json:"updated_at"` // The point in time this release of the plugin was added to the Plugin Marketplace
+	Enterprise      bool                      `json:"enterprise,omitempty"` // Indicated if the plugin is an enterprise plugin
```

Comment:
Using `omitempty` ensures there is no diff in `plugins.json`. On the other hand, not omitting this field might be cleaner, as one can see at first glance, what type of plugin it is.

Reply:
Personally I like being explicit with fields and would want to get rid of `omitempty` here -- the `plugins.json` diff noise will only happen once so personally don't mind that. But 0/5 on this since it only affects `plugins.json`.

Reply:
:+1: for being explicit